app/wsdl2rest/pom.xml

```diff
@@ -0,0 +1,149 @@
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" 
+    xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>wsdl2rest-impl-fatjar</artifactId>
+    <name>Wsdl2Rest Fat Jar</name>
+    <version>0.1.3-SNAPSHOT</version>
+    <groupId>org.github.camel.tooling.wsdl2rest</groupId>
+
+    <properties>
+        <version.wsdl2rest>0.8.0.fuse-720034</version.wsdl2rest>
+        <!-- <version.wsdl2rest>0.7.0</version.wsdl2rest> -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+      <dependency>
+        <groupId>org.jboss.fuse.wsdl2rest</groupId>
+        <artifactId>wsdl2rest-impl</artifactId>
+        <version>${version.wsdl2rest}</version>
+      </dependency>
+    </dependencies>
+
+    <build>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+				<version>2.0.4.RELEASE</version>
```

Comment:
latest version is now 2.1.0.RELEASE, might worth upgrading to latest and greatest on initial commit